src/runners/helpers/db.py

```diff
@@ -344,3 +349,13 @@ def record_metadata(metadata, table, e=None):
 
     except Exception as e:
         log.error(f"{record_type} metadata failed to log.", e)
+
+
+def ingest_request_failed(table, r, timestamp):
```

Comment:
functions should be called after what they do, and events (in pub/sub model) after what it is that happened. how about `record_failed_ingestion` here?

Reply:
done